### DISCUSSION TITLE
spoofdpi stopped working

### DISCUSSION THREAD

**Discussion by never2muchhhhhhhh** [xvzc/SpoofDPI#267] [Q&A]
On October 4, the program worked great, but at one point it simply stopped loading YouTube
I used the following commands:
curl -fsSL https://raw.githubusercontent.com/xvzc/SpoofDPI/main/install.sh | bash -s darwin-amd64
export PATH=$PATH:~/.spoof-dpi/bin
spoof-dpi
If anyone has encountered a similar problem or knows a way to solve it, please let me know. I will be immensely grateful

**Comment by Vroit** [comment]
The same here)

**Comment by Montager01** [comment]
<img width="332" alt="Снимок экрана 2024-10-13 в 09 44 11" src="https://github.com/user-attachments/assets/fc8fb81f-4210-4461-89b0-7fbfceb2b4d9">
<img width="404" alt="Снимок экрана 2024-10-13 в 09 44 18" src="https://github.com/user-attachments/assets/59e596b1-08bd-4753-823f-cd3811a6cc15">
it started on October 13 with the following value specified in the settings  spoof-dpi --enable-doh 8.8.8.8 --window-size 1
